libs/ledger-live-common/src/hw/actions/uninstallAllApps.ts

```diff
@@ -1,125 +0,0 @@
-import { Observable } from "rxjs";
-import { scan, tap } from "rxjs/operators";
-import { useCallback, useEffect, useState } from "react";
-import { log } from "@ledgerhq/logs";
-import type { DeviceInfo } from "@ledgerhq/types-live";
-import { useReplaySubject } from "../../observable";
-import type { UninstallAllAppsEvent, Input as UninstallAllAppsInput } from "../uninstallAllApps";
-import type { Action, Device } from "./types";
-import { currentMode } from "./app";
-import { getImplementation } from "./implementations";
-
-type State = {
-  isLoading: boolean;
-  unresponsive: boolean;
-  requestQuitApp?: boolean;
-  uninstallAppsRequested?: boolean;
-  appsUninstalled: boolean;
-  device: Device | null | undefined;
-  deviceInfo: DeviceInfo | null | undefined;
-  error: Error | null | undefined;
-};
-type StateWithRetry = State & { onRetry: () => void };
-
-type RemoveImageAction = Action<unknown, StateWithRetry, boolean>;
-
-const mapResult = ({ appsUninstalled }: State) => appsUninstalled;
-
-type Event =
-  | UninstallAllAppsEvent
-  | {
-      type: "error";
-      error: Error;
-    }
-  | {
-      type: "deviceChange";
-      device: Device | null | undefined;
-    };
-
-const getInitialState = (device?: Device | null | undefined): State => ({
-  isLoading: !!device,
-  requestQuitApp: false,
-  unresponsive: false,
-  device,
-  deviceInfo: null,
-  error: null,
-  uninstallAppsRequested: false,
-  appsUninstalled: false,
-});
-
-const reducer = (state: State, e: Event): State => {
-  switch (e.type) {
-    case "unresponsiveDevice":
-      return { ...state, unresponsive: true, isLoading: false };
-
-    case "deviceChange":
-      return getInitialState(e.device);
-
-    case "error":
-      return {
-        ...state,
-        error: e.error,
-        isLoading: false,
-      };
-    case "uninstallAppsPermissionRequested":
-      return {
-        ...state,
-        unresponsive: false,
-        uninstallAppsRequested: true,
-        isLoading: false,
-      };
-    case "appsUninstalled":
-      return {
-        ...state,
-        unresponsive: false,
-        uninstallAppsRequested: false,
-        isLoading: false,
-        appsUninstalled: true,
-      };
-  }
-};
-
-export const createAction = (
```

Comment:
Is it possible to keep this one ? It's associated command is `libs/ledger-live-common/src/hw/uninstallAllApps.ts`, which is only used in our cli. But Juan created 4 months ago just before leaving and we might use it

Reply:
done